mysql-test/t/persisted_variables.test

```diff
@@ -384,7 +384,9 @@ SELECT @@global.general_log, @@global.autocommit,
 
 SELECT @@global.innodb_buffer_pool_size;
 --remove_file $MYSQLD_DATADIR/mysqld-auto.cnf
-SET PERSIST innodb_buffer_pool_size=10*1024*1024;
+# In Percona Server, resize to 20MB instead of 10MB to stay compatible
```

Comment:
optional: instead of 'compatible', would be nice if you can mention about the hang with buf pool size < 20MB

Reply:
not doing this as discussed on Slack